index.html

```diff
@@ -1661,37 +1661,41 @@
       The algorithm to <dfn data-lt="microdata to rdfa">convert a Microdata <a>item</a> to RDFa</dfn> is as follows:</p>
 
     <ol>
-      <li>Replace the <a>itemscope</a> attribute with a <code>vocab</code> attribute,
-        whose value is the <a>vocabulary identifier</a> for the item.</li>
-      <li>Let <var>local type</var> be the difference between the <a>vocabulary identifier</a> and the first
-        value of <a>itemtype</a>. Replace the <a>itemtype</a> attribute with a <code>typeof</code> attribute
-        whose value is <var>local type</var>.</li>
-      <li>If <a data-lt="split a string on spaces">splitting the value on spaces</a> for the <a>itemtype</a> attribute
-        results in more than one value,
-        then for each value <var>extra type</var> of <a>itemtype</a> after the first,
-        add an empty <code>span</code> child element to the element that represents the item,
-        with the attribute <code>rel="rdf:type"</code>
-        and a <code>resource</code> attribute whose value is <var>extra type</var>.</li>
-      <li>Change each <a>itemprop</a> attribute to a <code>property</code> attribute with the same value</li>
-      <li>Change each <a>itemid</a> attribute to an <code>about</code> attribute with the same value</li>
-      <li>For the value of each <a>itemref</a> attribute, for each value <var>reference</var>
-        that is a result of <a data-lt="split a string on spaces">splitting the value on spaces</a> follow the following substeps:
+      <li>For each element having an <a>itemscope</a> attribute:
         <ol>
-          <li>Add an element as the immediate parent of the element with <code>id</code> <var>reference</var>,
-            with a <code>vocab</code> attribute whose value is the <a>vocabulary identifier</a> for the item.
-            <p class="note">The choice of element is left to the author, to provide sufficient flexibility
-            to avoid unwanted changes in the rendering of the content.</p>
+          <li>Replace the <a>itemscope</a> attribute with a <code>vocab</code> attribute, whose value is the <a>vocabulary identifier</a> for the item.</li>
+          <li>Replace any <a>itemtype</a> attribute with a <code>typeof</code> attribute whose value is taken from the values of <a>itemtype</a> where each type is made relative to <a>vocabulary identifier, ensuring that the element has a <code>typeof</code> attribute with at least an empty value.</a>.
```

Comment:
Making these relative doesn’t really add anything, and any that are already relative will be resolved against in-scope typeof.